WordPress/src/main/java/org/wordpress/android/ui/posts/EditPostActivity.java

```diff
@@ -2279,22 +2279,21 @@ public void onMediaChanged(OnMediaChanged event) {
      * Starts the upload service to upload selected media.
      */
     private void startUploadService(MediaModel media) {
-        final ArrayList<MediaModel> mediaList = new ArrayList<>();
-        if (MediaUploadState.QUEUED.toString().equals(media.getUploadState())) {
-            mediaList.add(media);
+        // make sure we only pass items with the QUEUED state to thee UploadService
```

Comment:
Minor nit but I have to point it out: should be `the` not `thee`.

Reply:
oops sorry afa25cb